RELEASE_NOTES.md

```diff
@@ -9,12 +9,22 @@
 [//]: # ( - Configurations public constructor are deprecated, please use each Configuration's builder to make a Configuration object)
 
 ## New
+- We added a [UI customization guide](https://github.com/Adyen/adyen-android/blob/develop/docs/UI_CUSTOMIZATION.md) for the Android SDK.
 
+## Improved
+- The SDK now uses JSON Web Encryption (JWE) with RSA OAEP 256 and AES GCM 256 for encryption. You do not need to make any changes to your integration.
```

Comment:
Nit: I'm not sure if we usually refer to it as the "SDK" or "library", worth double checking?

Reply:
The technical writer suggested to use `The integration`. How does that sound?

Reply:
👌